### ISSUE TITLE
Feature: Please consider "Zope Public License"

### ISSUE THREAD

**Issue by fhg-isi** [FHPythonUtils/LicenseCheck#43]
### Is your feature request related to a problem? Please describe

I use the library waitress, having a "Zope Public License":

https://github.com/Pylons/waitress
https://github.com/Pylons/waitress/blob/main/LICENSE.txt

licensecheck yields 

WARN: 'ZOPE PUBLIC LICENSE' License not identified so falling back to NO_LICENSE 

### Describe the solution you'd like

Consider the license

### Describe alternatives you've considered

Support of a whitelist, so that I can alter the behavior of licensecheck by configuration.

**Comment by fhg-isi** [comment]
As for the alternative, just saw the --ignore-licenses option:
https://github.com/FHPythonUtils/LicenseCheck#help

**Comment by NicolaDonelli** [comment]
@fhg-isi you could try to use the `ignore-packages` flag

```
[tool.licensecheck]
using = "requirements:requirements/requirements.txt"
ignore_packages = ['requests', 'waitress', 'watchdog']
zero = true
```

**Comment by FredHappyface** [comment]
Hopefully the ignore-licenses has been fixed. But yeah the ignore-packages is another workaround you may want to use. Also depending on how you use licensecheck you may just want to make a note when using it / for the development team? Of course this is a different story for CICD. I'll be sure to investigate this further and hopefully can confirm a fix for that behaviour

Not sure when I'll be supporting Zope public license yet, there are a few bits of refactoring I'd like to do to support unknown compatibility states and improve compatibility management

**Comment by fhg-isi** [comment]
>Hopefully the ignore-licenses has been fixed

Nope. But ignoring the individual package(s) that use that license worked as a workaround.
